pkg/apimachinery/tests/api_meta_help_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package meta_test
+package tests
```

Comment:
`api.List` is generic as well. `Pod` and `PodList` can be replaced with some other types. Then this test can go to apimachinery as well.